appcli/app.go

```diff
@@ -18,7 +18,7 @@ func newBaseApp() *cli.App {
 	app.HideVersion = true
 	app.HideHelp = true
 	app.EnableBashCompletion = true
-	app.UseShortOptionHandling = true
+	// app.UseShortOptionHandling = true
```

Comment:
I actually am using a fork of urfave/cli, since I've had a stale PR open for supporting this option for a while now. 

The `go.mod` file specifies the version that makes this line work, so to get that locally, you'll need to either work off the `GOPATH` or set `GO111MODULE=on`

Reply:
Got it! I'm not familiar with Go, didn't know about that :)